new_scripts/Ca_events_GCaMP2.m

```diff
@@ -0,0 +1,67 @@
+%This script is meant to run on ONE GENOTYPE'S DATA at a time. 
```

Comment:
Seems like a variation of Ca_events_GCaMP.m. Could be resolved to keep one version.